Cargo.lock

```diff
@@ -5588,6 +5585,10 @@ dependencies = [
  "vcpkg",
 ]
 
+[[package]]
+name = "lightclient"
+version = "0.1.0"
+
```

Comment:
This was somehow added by accident, it is not used and doesn't seem to correspond to any real crate

Reply:
Strange. There was indeed a lightclient-proxy crate in the beginning but was removed and I did ran `cargo update` multiple times. 

Anyway, let me send a PR to remove it.

Reply:
Already gone on `main` :wink:

Reply:
Thank you. :)